### ISSUE TITLE
[Feature : Flat Tour Book View] Filter tours which are selected/not selected

### ISSUE THREAD

**Issue by wolfgang-ch** [mytourbook/mytourbook#1050]
### What is missing ?

When tours are selected in other views, e.g. 2D map, show only these selected tours.

This filter is helpful to debug this request https://github.com/mytourbook/mytourbook/issues/966 which is the reason why it was created.

#### Limits
This filter works only for the flat tour layout.

**Comment by wolfgang-ch** [comment]
@FJBDev @telemaxx 

Can you test this new feature which is now in the main branch.

The tour collection filter is available only for the flat view. When the action button is selected, then the filter is active but the slideout is also displayed when the action button is not selected which helps to see the number of tours.

A tour collection can be created by selecting tours in views, other than the `Tour Book` view, e.g. in the 2D map bread crumb.

Do you understand this feature and how to use it, I changed the wording several times,

![mt-tourbook-filter 1](https://user-images.githubusercontent.com/1283445/231458668-5a321ec5-d835-4c92-a620-3fe675f93297.png)


![mt-tourbook-filter 2](https://user-images.githubusercontent.com/1283445/231458675-270d8819-168c-45ee-807a-29b070b80e2b.png)

**Comment by telemaxx** [comment]
i merged and tried, but no success.

![grafik](https://user-images.githubusercontent.com/22706487/231510566-ac3300e8-da74-47d7-bde1-6639b7e88a38.png)

![grafik](https://user-images.githubusercontent.com/22706487/231511023-ef563c41-a970-4a16-9af6-a5f568b2ac98.png)

not sure, what i have to do.

**Comment by FJBDev** [comment]
> not sure, what i have to do.

You will need to add the `TableCombo` plugin as below:
I added it when I implemented the new [Air Quality feature](https://github.com/mytourbook/mytourbook/pull/1048)

![image](https://user-images.githubusercontent.com/25235665/231513610-0988b3bb-2118-4792-9457-672c46ecbc95.png)

**Comment by telemaxx** [comment]
I think i have to install something in eclipse.
in my eclipse 2022-06 (2022-09 i had trouble in the past, but not remember which)
it looks like this:

![grafik](https://user-images.githubusercontent.com/22706487/231519888-8d293063-d073-4db8-9505-e0a9188d5639.png)

![grafik](https://user-images.githubusercontent.com/22706487/231520222-b6766e95-c793-45c0-9d15-650c02df973b.png)

![grafik](https://user-images.githubusercontent.com/22706487/231520470-61852b9f-fd58-42fd-9f39-9556f87007db.png)

**Comment by FJBDev** [comment]
> I think i have to install something in eclipse. in my eclipse 2022-06 (2022-09 i had trouble in the past, but not remember which) it looks like this:
> 

Yes, you will need to install [Nebula](https://www.eclipse.org/nebula/downloads.php)

**Comment by telemaxx** [comment]
Thank you @FJBDev , now i am close.
Nebula is OK now.

But the models from @wolfgang-ch are missing.

![grafik](https://user-images.githubusercontent.com/22706487/231547236-3719d986-e45d-4ff8-8c15-4246d4274afc.png)

i investigate tomorrow.

**Comment by FJBDev** [comment]
I tested this feature but don't see the map updated when I select the different slideout options:
Maybe I am confused by the slideout options and what they do on the 2d map


![image](https://user-images.githubusercontent.com/25235665/231576452-d4ebec33-b497-4deb-bac9-957946448fc2.png)

**Comment by FJBDev** [comment]
> Thank you @FJBDev , now i am close. Nebula is OK now.
> 
> But the models from @wolfgang-ch are missing.
> 
> 
> i investigate tomorrow.

You will need to import [this new plugin](https://github.com/mytourbook/mytourbook/tree/main/bundles/net.tourbook.model) into Eclipse

**Comment by wolfgang-ch** [comment]
> I tested this feature but don't see the map updated when I select the different slideout options: Maybe I am confused by the slideout options and what they do on the 2d map
> 
> ![image](https://user-images.githubusercontent.com/25235665/231576452-d4ebec33-b497-4deb-bac9-957946448fc2.png)

The tour collection filter (slideout) is applied to the tours which are displayed in the tour book view. First you have to select/collect tours outside of the tour book view, like in the 2D map bread crumb or within this map.

>...Maybe I am confused ...

That's what I somehow expected and asked for this review. Maybe we can change the wording for what this tool is about

**Comment by telemaxx** [comment]
now it is running, thanks @FJBDev
(i deleted my question about eclipse setup, to keep this threat clean)

The 1st use of this new function is a bit hard.
important to know is Tourbook must be in flatview.

"Link with other views" is not clear to me yet.

but slowly i understand the idea behind the collections.
eg tagging exporting multiple tours, etc.

finding good self explaining wordings is not easy here.

**Comment by wolfgang-ch** [comment]
> "Link with other views" is not clear to me yet.

When "Link with other views" is selected and you select a tour in another view, e.g. calendar view, then this tour is selected also in the tour book view

**Comment by FJBDev** [comment]
I was busy with my latest [PR](https://github.com/mytourbook/mytourbook/pull/1064) but I will resume testing your feature soon
